src/script/util/Logger.ts

```diff
@@ -17,15 +17,42 @@
  *
  */
 
-import {LogFactory, Logger} from '@wireapp/commons';
+import {LogFactory, Logger, Runtime} from '@wireapp/commons';
 
 const LOGGER_NAMESPACE = '@wireapp/webapp';
 
+function serializeArgs(args: any[]): any[] {
+  return args.map(arg => (typeof arg === 'object' && arg !== null ? JSON.stringify(arg) : arg));
+}
+
 function getLogger(name: string): Logger {
-  return LogFactory.getLogger(name, {
+  const logger = LogFactory.getLogger(name, {
     namespace: LOGGER_NAMESPACE,
     separator: '/',
   });
+
+  if (Runtime.isDesktopApp()) {
+    return {
+      ...logger,
+      debug: (...args: any[]): void => {
+        logger.debug(...serializeArgs(args));
+      },
+      error: (...args: any[]): void => {
+        logger.error(...serializeArgs(args));
+      },
+      info: (...args: any[]): void => {
+        logger.info(...serializeArgs(args));
+      },
+      log: (...args: any[]): void => {
+        logger.log(...serializeArgs(args));
+      },
+      warn: (...args: any[]): void => {
+        logger.warn(...serializeArgs(args));
+      },
+    };
+  }
+
+  return logger;
 }
```

Comment:
No a big thing, but `unknown` wouldn't be better for this case?

Reply:
I've followed the signature of the original logdown library:

<img width="716" alt="image" src="https://github.com/user-attachments/assets/660eb942-9792-4248-afb2-39d36ae9f90b">

Reply:
sure, it's not a big deal either way 👌